[IMP] Expenses: updating for new 16 design (#4702)
odoo/documentation: content/applications/finance/expenses.rst

.. seealso::
   `Odoo Expenses: product page <https://www.odoo.com/app/expenses>`_

Set expense categories
======================

The first step to track expenses is to configure the different types of expenses for the company
(managed as *expense categories* in Odoo). Each category can be as specific or generalized as
needed. Go to :menuselection:`Expenses app --> Configuration --> Expense Categories` to view the
current expensable categories in a default list view.

.. image:: expenses/categories.png
   :align: center
   :alt: Set expense costs on products.

To create a new expense category , click :guilabel:`New`. A *product* form will appear, with the
description field labeled :guilabel:`Product Name`.

.. note::
   Expense *categories* are managed like *products* in Odoo. The expense category form follows the
   standard *product* form in Odoo, and the information entered is similar. Expense *products* will
   be referred to as expense *categories* throughout this document since the main menu refers to

>    these as :guilabel:`Expense Categories`.

Author: OK- for this and the edit above, since the form is a product form, and the name of the "thing" you are creating is a 'category' instead of a 'product' - there was room for confusion, which is why I did all the italics (because it's crazy to call a category a product, or have it function like one but call it the other).

:guilabel:`Cost` field. Otherwise, leave the :guilabel:`Cost` set to `0.00`, and employees will
report the actual cost when submitting an expense report.

.. note::
   The :guilabel:`Cost` field is always visible on the expense category form, but the
   :guilabel:`Sales Price` field is *only* visible if the :guilabel:`Sales Price` is selected under

>    the :guilabel:`Re-Invoice Expenses` section. Otherwise, the :guilabel:`Sales Price` field is hidden.

Reviewer: Line over 100 chars
```suggestion
   the :guilabel:`Re-Invoice Expenses` section. Otherwise, the :guilabel:`Sales Price` field is
   hidden.
```

  expense, enter it in this field.
- :guilabel:`Expense Date`: Using the calendar module, enter the date the expense was incurred. Use
  the :guilabel:`< (left)` and :guilabel:`> (right)` arrows to navigate to the correct month, then
  click on the specific day to enter the selection.
- :guilabel:`Account`: Select the expense account that this expense should be logged on from the
  drop-down menu.
- :guilabel:`Customer to Reinvoice`: If the expense is something that should be paid for by a
  customer, select the :abbr:`SO (Sales Order)` and customer that will be invoiced for this expense
  from the drop-down menu. All sales orders in the drop-down menu list both the :abbr:`SO (Sales
  Order)` as well as the company the sales order is written for, but after the expense is saved, the
  customer name disappears and only the :abbr:`SO (Sales Order)` is visible on the expense.

  .. example::
     A customer wishes to have an on-site meeting for a custom garden (design and installation) and
     agrees to pay for the expenses associated with it (such as travel, hotel, meals, etc.). All
     expenses tied to that meeting would indicate the *sales order* for the custom garden (which

>      also references the customer) as the :guilabel:`Customer to Reinvoice`.

Author: ok, I initially did because that's a weird thing to reference - you would expect the customer name to be the reference, not a sales order, but I understand- and agree, and changed it =)

- :guilabel:`Notes...`: If any notes are needed in order to clarify the expense, enter them in the
  notes field.

.. image:: expenses/expense-filled-in.png
   :align: center
   :alt: A filled in expense form for a client lunch.

Attach a receipt
~~~~~~~~~~~~~~~~

After the expense is created, the next step is to attach a receipt. Click the :guilabel:`Attach
Receipt` button, and a file explorer appears. Navigate to the receipt to be attached, and click
:guilabel:`Open`. The new receipt is recorded in the chatter, and the number of receipts will appear
next to the :guilabel:`📎 (paperclip)` icon beneath the expense form. More than one receipt can be
attached to an individual expense, as needed. The number of receipts attached to the expense will be
noted on the paperclip icon.

.. image:: expenses/receipt-icon.png
   :align: center
   :alt: Attach a receipt and it appears in the chatter.

Create new expenses from a scanned receipt
------------------------------------------

Rather than manually inputting all of the information for an expense, expenses can be created by
scanning a PDF receipt.

> First, in the main *Expenses* app dashboard view (this view can also be accessed from

Reviewer 2: this might get a gui bc it's the name of a dashboard, no?
```suggestion
First, in the main :guilabel:`Expenses` app dashboard view (this view can also be accessed from
```

> :menuselection:`Expenses app --> My Expenses --> My Expenses to Report`), click :guilabel:`Scan`,

Reviewer 2: `My Expenses to Report` doesn't appear to be the name of the menu link...
```suggestion
:menuselection:`Expenses app --> My Expenses --> My Expenses`), click :guilabel:`Scan`,
```

Author: Oof- thanks for that catch- that was from the other versions, yikes! I thought I got them all- nope =)
Fixed now

> and a file explorer pops up. Navigate to the receipt to be uploaded, click on it, then click

Reviewer 2: need a conjunction here
```suggestion
and a file explorer pops up. Navigate to the receipt to be uploaded, click on it, and then click
```

next to each expense to add, then click :guilabel:`Select`. The items now appear on the report that
was just created. If a new expense needs to be added that does *not* appear on the list, click
:guilabel:`New` to create a new expense and add it to the report.

.. image:: expenses/add-an-expense-line.png
   :align: center
   :alt: Add more expenses to the report before submitting.

.. note::
   Expense reports can be created in one of three places:

   #. Go to the main :menuselection:`Expenses` app dashboard (also accessed by going to
      :menuselection:`Expenses app --> My Expenses --> My Expenses`)
   #. Go to :menuselection:`Expenses app --> My Expenses --> My Reports`
   #. Go to :menuselection:`Expenses app --> Expense Reports`

   In any of these views, click :guilabel:`New` to create a new expense report.

.. _expenses/submit:

Submit an expense report
------------------------

When an expense report is completed, the next step is to submit the report to a manager for
approval. Reports must be individually submitted, and cannot be submitted in batches. Open the
specific report from the list of expense reports (if the report is not already open). To view all

> your expense reports, go to :menuselection:`Expenses app --> My Expenses --> My Reports`.

Author: My logic was, that when you go to "my" reports, it is only "your "reports- but if you look at "all" reports (a different filter) and you are a manager, you see ALL reports- reports for other people you can approve, not just yours. So I thought it might be important to mention it was only "your" reports- even though I know that word does not belong in the doc .  But I took it out- hopefully people get it on their own, or from context =)

email. Odoo creates the expense by taking the information in the email subject and combining it with
the receipt.

To check an expense categories internal reference, go to :menuselection:`Expenses app -->
Configuration --> Expense Categories`. If an internal reference is listed on the expense category,
it is listed in the :guilabel:`Internal Reference` column.

.. image:: expenses/ref.png
   :align: center
   :alt: Internal reference numbers are listed in the main Expense Categories view.

> To add an internal reference on an expense category, click on the category to call up the form.

Reviewer 2: wording
```suggestion
To add an internal reference on an expense category, click on the category to open the form.
```

When viewing expense reports, there is a panel of filters to chose from on the left side. The three
categories that filters can be applied on are :guilabel:`Status`, :guilabel:`Employee`, and
:guilabel:`Company`. To view only expense reports with a particular status, click the checkbox next
to the status to display the expense reports with that status. Click the checkbox again to hide the
reports with that status. To view expense reports for a particular employee and/or company, click on
the employee name and/or company name in the :guilabel:`Employee` and :guilabel:`Company` sections.

Reports can be approved in two ways (individually or several at once) and refused only one way. To
approve multiple expense reports at once, remain in the list view. First, select the reports to
approve by clicking the check box next to each report, or click the box next to :guilabel:`Employee`
to select all reports in the list.

.. important::
   Only reports with a status of :guilabel:`Submitted` can be approved. It is recommended to only
   display the submitted reports by adjusting the status filter on the left side by only having the

>    :guilabel:`Submitted` filter checked off.

Reviewer 2: The idiom "checked off" I don't think works in this doc. It's usually in reference to to-do lists and productivity, as in, something was "done," not so much "enabled." As well idioms have a tendency to translate poorly into other languages so I'd be be more specific throughout the doc on every instance it shows up, e.g.:
```suggestion
   :guilabel:`Submitted` filter enabled.
```

Author: That makes a lot of sense- I changed all the check boxes where it applies to selecting filters/enabling items. I left the other "check" works when it was explaining how to select a few reports vs selecting the entire list (you'd click a different box for that). I *think* that makes sense- but if you'd like those parts changed as well, LMK.

Basically, "checked off" is gone, but "check the box" I kept.

   :align: center
   :alt: Approve multiple reports by clicking the checkboxes next to each report.

To approve an individual report, click on a report to go to a detailed view of that report. In this
view, several options are presented: :guilabel:`Approve`, :guilabel:`Report in Next Payslip`,
:guilabel:`Refuse`, or :guilabel:`Reset to draft`. Click :guilabel:`Approve` to approve the report.

If :guilabel:`Refuse` is clicked, a pop-up window appears. Enter a brief explanation for the refusal
in the :guilabel:`Reason to Refuse Expense` field, then click :guilabel:`Refuse`.

.. image:: expenses/refuse-expense.png
   :align: center
   :alt: Send messages in the chatter.

Team managers can easily view all the expense reports for their team members. While in the
:guilabel:`Expense Reports` view, click the drop-down arrow in the right-side of the search box, and
click on :guilabel:`My Team` in the :guilabel:`Filters` section. This presents all the reports for
the manager's team.

.. image:: expenses/my-team-filter.png
   :align: center
   :alt: Select the My Team filter.

.. note::
   If more information is needed, such as a receipt is missing, communication is easy from the

>    chatter. In an individual report, simply click :guilabel:`Send message` to open the message text

Reviewer: Should _message_ be capitalized here? Is it on the UI?
```suggestion
   chatter. In an individual report, simply click :guilabel:`Send message` to open the message text
```

Author: It's the box that pops up that allows you to type in a message- but the box itself isn't labeled as anything, so I am using the term "message box" generically. I am not sure if there is a better word/description for it but that made the most sense to me. I left it lower case because that is not the proper name for it, and you don't see those words on the UI.

===========================

Once an expense report is approved, the next step is to post the report to the accounting journal.
To view all expense reports, go to :menuselection:`Expenses --> Expense Reports`. To view only the

> expense reports that have been approved and need to be posted, adjust the filters on the left
> side so that only the :guilabel:`Approved` status is active (checked off).

Reviewer: Early line break
```suggestion
expense reports that have been approved and need to be posted, adjust the filters on the left side
so that only the :guilabel:`Approved` status is active (checked off).
```

post multiple expense reports at once, remain in the list view. First, select the reports to post by
clicking the check box next to each report, or click the box next to :guilabel:`Employee` to select
all reports in the list. Next, click :guilabel:`Post Entries`.

.. image:: expenses/post-entries.png
   :align: center
   :alt: Post multiple reports at a time from the Expense Reports view, with the Approved filter.

To post an individual report, click on a report to go to the detailed view of that report. In this
view, several options are presented: :guilabel:`Post Journal Entries`, :guilabel:`Report In Next
Payslip`, :guilabel:`Refuse`, or :guilabel:`Reset to Draft`. Click :guilabel:`Post Journal Entries`
to post the report.

If :guilabel:`Refuse` is clicked, a pop-up window appears. Enter a brief explanation for the refusal
in the :guilabel:`Reason to Refuse Expense` field, then click :guilabel:`Refuse`. Refused reports
can be viewed by going to :menuselection:`Expenses app --> Expense Reports`, then adjusting the
filters on the left so that only :guilabel:`Refused` is selected. This will only show the refused
expense reports.

> .. Important::

Reviewer 2: ```suggestion
.. important::
```

and :guilabel:`Payment Date` fields, a :guilabel:`Recipient Bank Account`, :guilabel:`Amount`, and
:guilabel:`Memo` field appear. Select the employee's bank account from the drop-down menu to direct
deposit the payment to their account. When all other selections are correct, click :guilabel:`Create
Payment` to send the payment to the employee.

.. image:: expenses/two-payment-posting-options.png
   :align: center
   :alt: Different options appear when registering an individual expense report versus multiple
         expense reports at once.

Re-invoice expenses to customers
================================

If expenses are tracked on customer projects, expenses can be automatically charged back to the
customer. This is done by creating an expense, referencing the :abbr:`SO (Sales Order)` the expense
should be added to, and then creating the expense report. Next, managers approve the expense report,
and the accounting department posts the journal entries. Finally, once the expense report is posted
to a journal, the expense(s) appears on the :abbr:`SO (Sales Order)` that was referenced. The sales
order can then be invoiced, thus invoicing the customer for the expense.

Setup
-----

> First, specify the invoicing policy for each expense category. Go to :menuselection:`Expenses
> app --> Configuration --> Expense Categories`. Click on the expense category to open the expense

Reviewer: Early line break
```suggestion
First, specify the invoicing policy for each expense category. Go to :menuselection:`Expenses app
--> Configuration --> Expense Categories`. Click on the expense category to open the expense
```

category form. Under the :guilabel:`Invoicing` section, click the radio button next to the desired
selection for :guilabel:`Re-Invoicing Expenses`. Options are :guilabel:`None`, :guilabel:`At cost`,
and :guilabel:`Sales price`.

:guilabel:`Re-Invoicing Expenses`:

- :guilabel:`None`: Expense category will not be re-invoiced.
- :guilabel:`At cost`: Expense category will invoice expenses at their real cost.
- :guilabel:`At sales price`: Expense category will invoice the price set on the sale order.

Create an expense
-----------------

First, when :ref:`creating a new expense <expenses/new>`, the correct information needs to be
entered in order to re-invoice a customer. Select the *sales order* the expense will appear on in
the :guilabel:`Customer to Reinvoice` section, from the drop-down menu. Next, select the
:guilabel:`Analytic Account` the expense will be posted to. After the expense(s) are created, the
expense report needs to be :ref:`created <expenses/report>` and :ref:`submitted <expenses/submit>`
as usual.

.. image:: expenses/reinvoice-expense.png
   :align: center
   :alt: Ensure the customer to be invoiced is called out on the expense.

.. important::

>    Selecting a :guilabel:`Customer to Reinvoice` when creating an expense is critical, since this
>    is what causes automatic invoicing after an expense report is approved. The :guilabel:`Customer
>    to Reinvoice` field can be modified *until an expense report is* **approved**, then the field is
>    no longer able to be modified.

Reviewer: Early line breaks
```suggestion
   Selecting a :guilabel:`Customer to Reinvoice` when creating an expense is critical, since this is
   what causes automatic invoicing after an expense report is approved. The :guilabel:`Customer to
   Reinvoice` field can be modified *until an expense report is* **approved**, then the field is no
   longer able to be modified.
```
